R/plot.R

```diff
@@ -1523,8 +1535,17 @@ plot.netdx <- function(x, type = "formation", method = "l", sims, stats,
            currently available", call. = FALSE)
     }
 
+    if (missing(duration.imputed)) {
+      duration.imputed <- TRUE
+    }
```

Comment:
If this has a universal default, it can be specified directly in the parameter list above instead of like this.

Reply:
Done.